### PR TITLE
chore: set up sqlite

### PR DESCRIPTION
**Pull request by martin-sweeny**
This PR adds sqlite integration to the project. Run `setup.py` if you're using a fresh db with no tables

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -4,22 +4,15 @@
 import asyncio
 import auxfunctions as aux
 import discord
-import math
```
Comment (slocknad) [inline bot.py:7]: We need the math module. Why are you removing it?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from dotenv import dotenv_values
+import sqlite3
+
+config = dotenv_values('.env')
+
+conn = sqlite3.connect(config['SQLITE_DB_FILE'])
+cur = conn.cursor()
+
+cur.execute('''CREATE TABLE Triggers
+               (name text, content text, embed text)''')
```
Comment (slocknad) [inline setup.py:10]: We should restructure this...
The initial idea was name, content, and a boolean indicating if it's an embed or not. But I feel like there might be better ways to do this

Comment (slocknad) [reply setup.py:10]: To be fair, I'm not even sure to what point it'll even be necessary to create embed triggers. We could just remove that altogether

Comment (martin-sweeny) [reply setup.py:10]: I'll update this file to be more generic and then we can remove triggers in a new PR

Comment (martin-sweeny) [reply setup.py:10]: How does that sound?

File: bot.py (hunk 1 above)
Comment (martin-sweeny) [reply bot.py:7]: image

Comment (martin-sweeny) [reply bot.py:7]: I dont remember doing that

File: setup.py (hunk 2 above)
Comment (martin-sweeny) [reply setup.py:10]: > We should restructure this...
> The initial idea was name, content, and a boolean indicating if it's an embed or not. But I feel like there might be better ways to do this

Coming back to this, I think it's better if we edit this script in the same PR we'd update or remove the embed triggers feature. For now, the script sets up the minimum db schema we need, right?

Comment (slocknad) [reply setup.py:10]: Yeah sure! Whatever you feel is best! It's just that until now, in the server history, we've only had one embed trigger... one that doesn't even exist anymore

Comment (martin-sweeny) [reply setup.py:10]: @slocknad wanna approve it then

**Review by slocknad — APPROVED** [review]

**Pull request merged**